README.md

```diff
@@ -0,0 +1,115 @@
+# CCF Backstage Plugin
+
+Understand how your cloud usage impacts our environment and what you can do about it.
+
+This plugin brings the functionallity of the open source project [Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org/) into Backstage. See [the CCF demo](https://demo.cloudcarbonfootprint.org/) and [the CCF docs](https://www.cloudcarbonfootprint.org/docs/overview) for more details.
+
+For more visibility into the work planned and happening, please visit the [project board](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/projects/1) or [create an issue.](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/new/choose)
+For feedback, questions, or if you’d like to collaborate on shaping the product direction, please reach out to [green-cloud@thoughtworks.com](mailto:green-cloud@thoughtworks.com).
+
+|Estimates | Forecast |
+|---|---|
+| ![Screenshot 2022-04-01 at 17 07 07](https://user-images.githubusercontent.com/8904624/161291252-c771f806-8ddf-4121-abf7-429217e64664.png) | ![Screenshot 2022-04-01 at 17 07 24](https://user-images.githubusercontent.com/8904624/161292172-e18d2894-45f1-4231-a353-035d821a9312.png) |
+
+| Recommendations | Carbon Intensity Map |
+|---|---|
+| ![Screenshot 2022-04-01 at 17 07 35](https://user-images.githubusercontent.com/8904624/161291044-309b7b62-106a-4254-bdd9-5b432cf35232.png) | ![Screenshot 2022-04-01 at 17 07 45](https://user-images.githubusercontent.com/8904624/161291047-1d093d5a-4883-448a-8d08-5180c8d867d7.png) |
+
+## Getting started
+
+If you haven't already, check out the [Backstage docs](https://backstage.io/docs/getting-started/) and create a Backstage application with
+```
+npx @backstage/create-app
+```
+
+Then, you will need to install and configure the CCF plugins for the frontend and the backend.
+
+### Frontend plugin
+
+Install:
+```bash
+cd packages/app
+yarn add @cloud-carbon-footprint/backstage-plugin-frontend
+```
+
+Add the plugin to `packages/app/src/App.tsx`:
+```jsx
+// import:
+import { CloudCarbonFootprintPage } from '@cloud-carbon-footprint/backstage-plugin-frontend'
+
+// add to routes:
+const routes = (
+  <FlatRoutes ...>
+  ...
+    <Route path="/cloud-carbon-footprint" element={<CloudCarbonFootprintPage />} />
+  </FlatRoutes>
+)
+```
+
+Pin webpack version in root `package.json`:
```

Comment:
this is a workaround. Without this, the plugin fails to load in dev mode with a strange error `returntrue is not defined`. from [this PR](https://github.com/mui/mui-x/issues/4490) i found the workaround to pin a version of webpack. 

I believe the real solution is to move from `@material-ui/data-grid@4.0.0-alpha.35` to `@mui/x-data-grid`, which involves a few changes in the RecommendationsPage, and potentially with upgrading other MUI libraries. I didn't test to see if other versions of `@material-ui/data-grid` don't have this problem